app/views/core/AuthModal.coffee

```diff
@@ -34,6 +34,12 @@ module.exports = class AuthModal extends ModalView
     @previousFormInputs = options.initialValues or {}
     @previousFormInputs.emailOrUsername ?= @previousFormInputs.email or @previousFormInputs.username
 
+    if options.loginMessage 
+      @loginMessage = options.loginMessage
+
+    if options.nextUrl
+      window.nextURL = options.nextUrl
```

Comment:
Should we clear `window.nextURL` in the destructor? Otherwise, if we happen to navigate away instead of logging in, then later when we log in we'll come back here.